.eslintrc

```diff
@@ -1,25 +1,29 @@
 {
+    "extends": "eslint:recommended",
     "parser": "babel-eslint",
     "plugins": ["babel"],
     "rules": {
-        "strict": 0,
-        "indent": [2, 4],
-        "quotes": [2, "single"],
-        "linebreak-style": [2, "unix"],
+        "consistent-return": 2,
+        "eqeqeq": 2,
         "semi": [2, "always"],
-        "no-console": 0,
-        "babel/generator-star-spacing": 1,
```

Comment:
I removed this because it doesn't behave correctly when `async` functions are used.